services/src/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -464,7 +464,8 @@ private CommandMessage buildDeleteFlowCommand(final String id, final String corr
     }
 
     private FlowPathPayload buildFlowPathPayload(List<FlowPathDto> paths, String flowId) {
-        FlowPathDto askedPathDto = paths.stream().filter(e -> e.getId().equals(flowId)).findAny().get();
+        FlowPathDto askedPathDto = paths.stream().filter(e -> e.getId().equals(flowId)).findAny()
+                .orElseThrow(() -> new IllegalStateException(format("Flow %s not found.", flowId)));
```

Comment:
incorrect message. Here we didn't found **path** for flow

Reply:
Fixed.